bot/src/main/kotlin/bot/horo/bot/command/ProfileExtension.kt

```diff
@@ -0,0 +1,145 @@
+package bot.horo.bot.command
+
+import com.kotlindiscord.kord.extensions.commands.Arguments
+import com.kotlindiscord.kord.extensions.commands.application.slash.publicSubCommand
+import com.kotlindiscord.kord.extensions.commands.converters.impl.user
+import com.kotlindiscord.kord.extensions.extensions.Extension
+import com.kotlindiscord.kord.extensions.extensions.publicSlashCommand
+import com.kotlindiscord.kord.extensions.types.respond
+import com.kotlindiscord.kord.extensions.utils.createdAt
+import dev.kord.common.Color
+import dev.kord.common.toMessageFormat
+import dev.kord.rest.builder.message.create.actionRow
+import dev.kord.rest.builder.message.create.embed
+import kotlinx.datetime.Clock
+
+class ProfileExtension : Extension() {
+    override val name: String = "horo.profile"
+    override val bundle: String = "horo.profile"
+
+    private class ProfileArguments : Arguments() {
+        val target by user {
+            name = "tag.arguments.user.name"
+            description = "tag.arguments.user.description"
+        }
+    }
+
+    override suspend fun setup() {
+        publicSlashCommand { //pass an instance of the ProfileArguments class as an argument
+            name = "name"
+            description = "description"
+            val theme = Color(202, 117, 201)
```

Comment:
If this colour is used more universally in the bot's branding, you should move it to a `Constants` file and give it a better name.

Reply:
I'll make constants like this soon since I'm planning on reusing some accent colours too